connectors/sources/sharepoint_online.py

```diff
@@ -761,12 +769,10 @@ async def get_docs(self, filtering=None):
                                 )
                                 yield list_item_attachment, attachment_download_func
 
-                        download_func = None
-
-                        yield list_item, download_func
```

Comment:
Question: Why is it not necessary to have these anyore?

Reply:
I moved it higher to line 751.

Also there's no `download_func` for it, so I just made it `yield list_item, None`